02-Custom-Login/app/controllers/home_controller.rb

```diff
@@ -0,0 +1,35 @@
+require "auth0"
+
+class HomeController < ApplicationController
+  attr_reader :client
+
+  def auth
+    v2_creds = { client_id: ENV['AUTH0_CLIENT_ID'],
+     token: '',
```

Comment:
I think you should use api_version: 1 to login with username and password where the token is no needed. Can you try?